project/Build.scala

```diff
@@ -106,7 +106,7 @@ object Build extends sbt.Build {
     }
   }
 
-  val commonSettings = Seq(
+  lazy val commonSettings = inThisBuild(baseSettings) ++ Seq(
```

Comment:
That is not ideal. It will replicate `baseSettings` as many times as there are projects.

It would be better to simply write `inThisBuild(Build.baseSettings)` as an independent line in `build.sbt`.